sensors/others/sensorhub-driver-f20/src/main/java/com/sample/impl/driver/f20/F20Config.java

```diff
@@ -0,0 +1,45 @@
+/***************************** BEGIN LICENSE BLOCK ***************************
+
+ The contents of this file are subject to the Mozilla Public License, v. 2.0.
+ If a copy of the MPL was not distributed with this file, You can obtain one
+ at http://mozilla.org/MPL/2.0/.
+
+ Software distributed under the License is distributed on an "AS IS" basis,
+ WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
+ for the specific language governing rights and limitations under the License.
+
+ The Initial Developer is Botts Innovative Research Inc. Portions created by the Initial
+ Developer are Copyright (C) 2025 the Initial Developer. All Rights Reserved.
+
+ ******************************* END LICENSE BLOCK ***************************/
+
+package com.sample.impl.driver.f20;
+
+import org.sensorhub.api.config.DisplayInfo;
+import org.sensorhub.api.sensor.SensorConfig;
+
+public class F20Config extends SensorConfig {
+
+    /**
+     * The unique identifier for the configured sensor (or sensor platform).
+     */
+    @DisplayInfo.Required
+    @DisplayInfo(desc = "Serial number or unique identifier")
+    public String serialNumber = "001";
+
+    @DisplayInfo.Required
+    @DisplayInfo(label = "MQTT Broker")
```

Comment:
Should this not have a field type:
@DisplayInfo.FieldType(DisplayInfo.FieldType.Type.REMOTE_ADDRESS)